cfme/tests/physical_infrastructure/test_redfish_physical_racks.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme.physical.provider.redfish import RedfishProvider
+
+pytestmark = [pytest.mark.provider([RedfishProvider], scope="function")]
+
+
+@pytest.fixture(scope="function")
+def physical_racks(appliance, provider, setup_provider_funcscope):
+    """Get and return the physical racks collection."""
+    yield appliance.collections.redfish_physical_racks.all(provider)
```

Comment:
Not sure if it makes sense to put this into a fixture, but okay

Reply:
@miha-plesko I agree here and would actually request that this is removed and this call made during the test. There's no setup occurring here, and I've recently made the same request to other PR's placing collections within fixtures. The test or any fixture requiring the collection and its entities will have the `appliance` fixture readily available from which to generate the list.